### DISCUSSION TITLE
the console does not work in the newest version

### DISCUSSION THREAD

**Discussion by Potato-eater** [pybricks/support#1129] [Q&A] [answered]
in the newest version of the pybricks beta IDE, the console does not work. I couldn't print anything.
![image](https://github.com/pybricks/support/assets/95858994/5fe128fc-3fee-4251-89fc-9b9e9ece0bb2)
it only shows as a black box with nothing inside.

**Comment by BertLindeman** [comment]
I do not see the complete program above, but what I do see would not print anything.
What do I miss?

Bert

**Reply by Potato-eater** [reply]
sorry, I put the wrong image. this is the program that I wanted to show
![image](https://github.com/pybricks/support/assets/95858994/cad9279a-268e-4b41-a8a4-7d249dc8864d)

**Comment by BertLindeman** [comment]
Ran this program

```python
from pybricks.hubs import ThisHub
from pybricks.parameters import Port

from pybricks import version
print(version)

```

on the current Pybricks beta with 

<html>
<body>


Connected to: | TechB.3.3.0b6
-- | --
Hub type: | Technic hub
Firmware: | v3.3.0b6


</body>
</html>

And indeed nothing shows up in the console.

Did the same with 
<html>
<body>


Connected to: | city.3.3.0b7
-- | --
Hub type: | City hub
Firmware: | v3.3.0b7


</body>
</html>
with the same result.

There was an issue that the console would not work if running a newer firmware on code.pybricks?

[EDIT]
```
About Pybricks Beta
Pybricks logo
MicroPython for LEGO® Powered Up smart hubs.

v3.3.0b7 (Pybricks Beta v2.2.0-beta.6)

Copyright (c) 2020-2023 The Pybricks Authors

LEGO® is a trademark of the LEGO Group of companies which does not sponsor, authorize or endorse this site.
```

**Comment by dlech** [comment]
Sheesh, I don't think my brain has been working very well lately. I totally forgot to fix that before I did the release. Will have a look right away.

**Reply by dlech** [reply, marked as answer]
Fix is now available (v2.2.0-beta.7).
